libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -36,7 +36,7 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @DisplayName: Fence Action
     // @Description: What action should be taken when fence is breached
     // @Values{Copter}: 0:Report Only,1:RTL or Land,2:Always Land,3:SmartRTL or RTL or Land,4:Brake or Land,5:SmartRTL or Land
-    // @Values{Rover}: 0:Report Only,1:RTL,2:Hold,3:SmartRTL,4:SmartRTL or Hold
+    // @Values{Rover}: 0:Report Only,1:RTL/Hold,2:Hold,3:SmartRTL/RTL/Hold,4:SmartRTL or Hold
```

Comment:
I think we should be consistent re "/" and "or".  We're mixing the two in a single set of options and users may wonder what the difference is.  I'd vote for using "or" because that's what we use in other places.

Reply:
done